# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/third_party_build/moz.build
# Commit: d838dc7d2bb8
# Full Hash: d838dc7d2bb89ab5c9c2e8e93895db1c9310560f
# Author: Dan Minor <dminor@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 4
# Description:
#   Bug 1654112 - Update include paths for moved upstream code. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D130059
# ==============================================================================

diff -r 587c40771588 -r d838dc7d2bb8 dom/media/webrtc/third_party_build/moz.build
--- a/dom/media/webrtc/third_party_build/moz.build	Fri Oct 23 13:49:51 2020 -0400
+++ b/dom/media/webrtc/third_party_build/moz.build	Fri Oct 23 14:02:54 2020 -0400
@@ -10,50 +10,38 @@
 include("/build/gn.mozbuild")
 
 webrtc_non_unified_sources = [
-    "../../../../third_party/libwebrtc/webrtc/common_audio/vad/vad_core.c",  # Because of name clash in the kInitCheck variable
-    "../../../../third_party/libwebrtc/webrtc/common_audio/vad/webrtc_vad.c",  # Because of name clash in the kInitCheck variable
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/acm2/codec_manager.cc",  # Because of duplicate IsCodecRED/etc
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/codecs/g722/g722_decode.c",  # Because of name clash in the saturate function
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/codecs/g722/g722_encode.c",  # Because of name clash in the saturate function
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/codecs/isac/fix/source/decode_plc.c",  # Because of name clash in the exp2_Q10_T function
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/codecs/isac/fix/source/lpc_masking_model.c",  # Because of name clash in the exp2_Q10_T function
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/codecs/isac/fix/source/pitch_filter.c",  # Because of name clash in the kDampFilter variable
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/codecs/isac/fix/source/pitch_filter_c.c",  # Because of name clash in the kDampFilter variable
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_coding/neteq/audio_vector.cc",  # Because of explicit template specializations
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/android/audio_manager.cc",  # Because of TAG redefinition
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/android/audio_record_jni.cc",  # Becuse of commonly named module static vars
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/android/audio_track_jni.cc",  # Becuse of commonly named module static vars
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/android/opensles_player.cc",  # Because of TAG redefinition
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/linux/audio_device_pulse_linux.cc",  # Because of LATE()
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/linux/audio_mixer_manager_pulse_linux.cc",  # Because of LATE()
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_device/win/audio_device_core_win.cc",  # Because of ordering assumptions in strsafe.h
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/aec/echo_cancellation.cc",  # Because of conflicts over 'near' on windows
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/aecm/aecm_core.cc",  # Because of the PART_LEN2 define
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/aecm/aecm_core_c.cc",  # Because of the PART_LEN2 define
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/aecm/aecm_core_mips.cc",  # Because of the PART_LEN2 define
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/aecm/aecm_core_neon.cc",  # Because of the PART_LEN2 define
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/aecm/echo_control_mobile.cc",  # Because of the PART_LEN2 define
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/agc/legacy/analog_agc.c",  # Because of name clash in the kInitCheck variable
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/beamformer/covariance_matrix_generator.cc",  # Because of needing to define _USE_MATH_DEFINES before including <cmath>
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/beamformer/covariance_matrix_generator.cc",  # Because of needing to define _USE_MATH_DEFINES before including <cmath>
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/beamformer/nonlinear_beamformer.cc",  # Because of needing to define _USE_MATH_DEFINES before including <cmath>
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/echo_cancellation_impl.cc",  # Because of name clash in the MapError function
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/echo_control_mobile_impl.cc",  # Because of name clash in the MapError function
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/echo_detector/normalized_covariance_estimator.cc",  # Because of kAlpha
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/gain_control_impl.cc",  # Because of name clash in the Handle typedef
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/noise_suppression_impl.cc",  # Because of name clash in the Handle typedef
-    "../../../../third_party/libwebrtc/webrtc/modules/audio_processing/rms_level.cc",  # Because of name clash in the kMinLevel variable
-    "../../../../third_party/libwebrtc/webrtc/modules/congestion_controller/trendline_estimator.cc",  # Because of name clash in kDeltaCounterMax
-    "../../../../third_party/libwebrtc/webrtc/modules/desktop_capture/win/screen_capturer_win_gdi.cc",  # Because base/logging.h uses #ifndef LOG before defining anything
-    "../../../../third_party/libwebrtc/webrtc/modules/remote_bitrate_estimator/remote_bitrate_estimator_single_stream.cc",  # Because of duplicate definitions of static consts against remote_bitrate_estimator_abs_send_time.cc
-    "../../../../third_party/libwebrtc/webrtc/modules/rtp_rtcp/source/flexfec_receiver.cc",  # Because of identically named functions and vars between flexfec_receiver.cc and flexfec_sender.cc in an anonymous namespaces
-    "../../../../third_party/libwebrtc/webrtc/modules/rtp_rtcp/source/rtcp_packet/tmmbn.cc",  # Because of identically named functions and vars between tmmbr.cc and tmmbn.cc in an anonymous namespaces
-    "../../../../third_party/libwebrtc/webrtc/modules/rtp_rtcp/source/rtcp_packet/tmmbr.cc",  # Because of identically named functions and vars between tmmbr.cc and tmmbn.cc in an anonymous namespaces
-    "../../../../third_party/libwebrtc/webrtc/modules/rtp_rtcp/source/ulpfec_generator.cc",  # Because of identically named constant kRedForFecHeaderLength in an anonymous namespace
-    "../../../../third_party/libwebrtc/webrtc/modules/video_capture/windows/device_info_ds.cc",  # Because of the MEDIASUBTYPE_HDYC variable
-    "../../../../third_party/libwebrtc/webrtc/modules/video_capture/windows/help_functions_ds.cc",  # Because of initguid.h
-    "../../../../third_party/libwebrtc/webrtc/modules/video_capture/windows/sink_filter_ds.cc",  # Because of the MEDIASUBTYPE_HDYC variable and initguid.h
-    "../../../../third_party/libwebrtc/webrtc/video/overuse_frame_detector.cc",  # Because of name clash with call_stats.cc on kWeightFactor
+    "../../../../third_party/libwebrtc/common_audio/vad/vad_core.c",                                  # Because of name clash in the kInitCheck variable
+    "../../../../third_party/libwebrtc/common_audio/vad/webrtc_vad.c",                                # Because of name clash in the kInitCheck variable
+    "../../../../third_party/libwebrtc/modules/audio_coding/codecs/isac/fix/source/decode_plc.c",   # Because of name clash in the exp2_Q10_T function
+    "../../../../third_party/libwebrtc/modules/audio_coding/codecs/isac/fix/source/lpc_masking_model.c",   # Because of name clash in the exp2_Q10_T function
+    "../../../../third_party/libwebrtc/modules/audio_coding/codecs/isac/fix/source/pitch_filter.c",   # Because of name clash in the kDampFilter variable
+    "../../../../third_party/libwebrtc/modules/audio_coding/codecs/isac/fix/source/pitch_filter_c.c", # Because of name clash in the kDampFilter variable
+    "../../../../third_party/libwebrtc/modules/audio_coding/neteq/audio_vector.cc",                   # Because of explicit template specializations
+    "../../../../third_party/libwebrtc/modules/audio_device/android/audio_manager.cc",                # Because of TAG redefinition
+    "../../../../third_party/libwebrtc/modules/audio_device/android/audio_record_jni.cc",             # Becuse of commonly named module static vars
+    "../../../../third_party/libwebrtc/modules/audio_device/android/audio_track_jni.cc",              # Becuse of commonly named module static vars
+    "../../../../third_party/libwebrtc/modules/audio_device/android/opensles_player.cc",              # Because of TAG redefinition
+    "../../../../third_party/libwebrtc/modules/audio_device/linux/audio_device_pulse_linux.cc",       # Because of LATE()
+    "../../../../third_party/libwebrtc/modules/audio_device/linux/audio_mixer_manager_pulse_linux.cc",# Because of LATE()
+    "../../../../third_party/libwebrtc/modules/audio_device/win/audio_device_core_win.cc",            # Because of ordering assumptions in strsafe.h
+    "../../../../third_party/libwebrtc/modules/audio_processing/aecm/aecm_core.cc",                   # Because of the PART_LEN2 define
+    "../../../../third_party/libwebrtc/modules/audio_processing/aecm/aecm_core_c.cc",                 # Because of the PART_LEN2 define
+    "../../../../third_party/libwebrtc/modules/audio_processing/aecm/aecm_core_mips.cc",              # Because of the PART_LEN2 define
+    "../../../../third_party/libwebrtc/modules/audio_processing/aecm/aecm_core_neon.cc",              # Because of the PART_LEN2 define
+    "../../../../third_party/libwebrtc/modules/audio_processing/aecm/echo_control_mobile.cc",         # Because of the PART_LEN2 define
+    "../../../../third_party/libwebrtc/modules/audio_processing/echo_control_mobile_impl.cc",         # Because of name clash in the MapError function
+    "../../../../third_party/libwebrtc/modules/audio_processing/echo_detector/normalized_covariance_estimator.cc", # Because of kAlpha
+    "../../../../third_party/libwebrtc/modules/audio_processing/gain_control_impl.cc",                # Because of name clash in the Handle typedef
+    "../../../../third_party/libwebrtc/modules/audio_processing/rms_level.cc",                        # Because of name clash in the kMinLevel variable
+    "../../../../third_party/libwebrtc/modules/desktop_capture/win/screen_capturer_win_gdi.cc",       # Because base/logging.h uses #ifndef LOG before defining anything
+    "../../../../third_party/libwebrtc/modules/remote_bitrate_estimator/remote_bitrate_estimator_single_stream.cc", # Because of duplicate definitions of static consts against remote_bitrate_estimator_abs_send_time.cc
+    "../../../../third_party/libwebrtc/modules/rtp_rtcp/source/flexfec_receiver.cc",                  # Because of identically named functions and vars between flexfec_receiver.cc and flexfec_sender.cc in an anonymous namespaces
+    "../../../../third_party/libwebrtc/modules/rtp_rtcp/source/rtcp_packet/tmmbn.cc",                 # Because of identically named functions and vars between tmmbr.cc and tmmbn.cc in an anonymous namespaces
+    "../../../../third_party/libwebrtc/modules/rtp_rtcp/source/rtcp_packet/tmmbr.cc",                 # Because of identically named functions and vars between tmmbr.cc and tmmbn.cc in an anonymous namespaces
+    "../../../../third_party/libwebrtc/modules/rtp_rtcp/source/ulpfec_generator.cc",                  # Because of identically named constant kRedForFecHeaderLength in an anonymous namespace
+    "../../../../third_party/libwebrtc/modules/video_capture/windows/device_info_ds.cc",              # Because of the MEDIASUBTYPE_HDYC variable
+    "../../../../third_party/libwebrtc/modules/video_capture/windows/help_functions_ds.cc",           # Because of initguid.h
+    "../../../../third_party/libwebrtc/modules/video_capture/windows/sink_filter_ds.cc",              # Because of the MEDIASUBTYPE_HDYC variable and initguid.h
 ]
 
 if CONFIG["MOZ_WIDGET_TOOLKIT"] == "gtk":